docs/reference/limitations/openstack.md

```diff
@@ -0,0 +1,40 @@
+# OpenStack service limitations
+
+
+## OpenStack users, groups, and roles
+
+### Domain admins
+
+No OpenStack user that you create while [enabling the OpenStack CLI](../../howto/getting-started/enable-openstack-cli.md) ever gets privileges exceeding administrative rights bound to a project.
+
+This means that you cannot use an OpenStack user to *create* a new project.
+You must do so via the {{gui}}.
```

Comment:
Or via Cleura API https://apidoc.cleura.cloud/#api-AccessControl_Openstack-OSCreateProject

Reply:
Fixed. Since we don't have a howto specifically on creating projects with the REST API, I have added a link to the API reference (which in turn links to apidoc.cleura.cloud).